container_images/tox/README.md

```diff
@@ -0,0 +1,63 @@
+A prow worker that executes Python tests using [tox](https://tox.readthedocs.io/).
+Check the Dockerfile for which Python versions are installed.
+
+### Interface
+
+Your job:
+1. Create a [tox configuration file](https://tox.readthedocs.io/en/latest/config.html) in your
+repository's test directory.
+   * Optionally, write test artifacts to the directory set in the `ARTIFACTS` environment variable.
+1. Configure prow with the test directory's path as the first positional argument.
+
+Prior to invoking, the runner will:
+1. Checkout your repository, mount it as a volume, and set its root as Docker's 
+[workdir](https://docs.docker.com/engine/reference/commandline/run/#set-working-directory--w).
+2. Mount a volume for artifacts, and set the `ARTIFACTS` environment variable to its path.
```

Comment:
This is my understanding of how prow sets up the environment, and I included it to help future readers and maintainers understand the execution environment. (It took me awhile to figure out which configurations were required, especially since we have cruft in [1]).

1. https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/test-infra/prowjobs